### PR TITLE
Publish to sonatype

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,2 +1,3 @@
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.0")
```
Comment (agemooij) [inline project/plugins.sbt:1]: Why do we need this if we use `sbt publish`?

**Comment by agemooij** [comment]
Looks good. I vaguely remember that this won't work until the correct credentials are encoded in the travis.yaml, right?

File: project/plugins.sbt (hunk 1 above)
Comment (raboof) [inline project/plugins.sbt:1]: publish for publishing snapshots, release plugin for executing a release procedure (which includes publishing a non-snapshot)?

**Comment by raboof** [comment]
You're probably right WRT credentials in travis. Will look into that.

File: project/plugins.sbt (hunk 1 above)
Comment (agemooij) [inline project/plugins.sbt:1]: Ah, of course

**Pull request merged**